rightmesh/contracts/Timelock.sol

```diff
@@ -62,13 +70,18 @@ contract Timelock is Ownable {
   }
 
   function availableForWithdrawal(address _address) public view returns (uint256) {
+    uint256 timelockEndTime = startTime.add(cliffDuration).add(gradualDuration);
+
     if (now < startTime.add(cliffDuration)) {
       return 0;
-    } else if (now > startTime.add(cliffDuration) && now < startTime.add(cliffDuration).add(gradualDuration)) {
+    } else if (now < timelockEndTime) {
       uint256 cliffTokens = (cliffReleasePercentage.mul(allocatedTokens[_address])).div(100);
-      uint256 divisor = gradualDuration.mul(100);
-      return (((((now.sub(startTime.add(cliffDuration))).mul(gradualReleasePercentage)).mul(allocatedTokens[_address])).div(divisor)).add(cliffTokens)).sub(withdrawnTokens[_address]);
-    } else if (now > startTime.add(cliffDuration).add(gradualDuration)) {
+      uint256 slopeTokens = (allocatedTokens[_address].mul(gradualReleasePercentage)).div(100);
+      uint256 timeAtSlope = now.sub(startTime.add(cliffDuration));
+      uint256 slopeTokensByNow = (slopeTokens.mul(timeAtSlope)).div(gradualDuration);
```

Comment:
More readable but this line now has two divisions in it (slopeTokens and .div(gradualDuration) when it could just have one.